hugolib/multilingual.go

```diff
@@ -0,0 +1,48 @@
+package hugolib
+
+import (
+	"github.com/spf13/cast"
+	"github.com/spf13/viper"
+)
+
+type Multilingual struct {
+	enabled bool
+	config  *viper.Viper
+
+	Languages []string
+}
+
+func (ml *Multilingual) GetString(key string) string { return cast.ToString(ml.Get(key)) }
+func (ml *Multilingual) GetStringMap(key string) map[string]interface{} {
+	return cast.ToStringMap(ml.Get(key))
+}
+
+func (ml *Multilingual) GetStringMapString(key string) map[string]string {
+	return cast.ToStringMapString(ml.Get(key))
+}
+
+func (ml *Multilingual) Get(key string) interface{} {
```

Comment:
I've looked into it for `paginatePath`, and it would require quite a bit of rework to have it there... because where `viper.Get("paginatePath")` is used is often outside the scope of a `Site`..

For now, I'd tolerate the fact that this piece isn't translated nor translateable.  I'm pushing some improved docs regarding this.